data/remnant missions.txt

```diff
@@ -249,7 +249,7 @@ mission "Remnant: Bounty"
 				"Korath Raider (Crippled)"
 		dialog "You have destroyed the Korath ship. You can now return to <planet> to collect your payment."
 	on complete
-		payment 200000
+		payment 300000
```

Comment:
If salvage 4 is 300k for a fail and 600k for a succeed, then I believe that bounty 1 should remain 200k and bounty 2 be 300k. Reason being that the ship in salvage 4 is the same ship as in bounty 2, yet right now the payment for failing salvage 4 is the same as taking out a bounty 1 ship.

Reply:
You suggested the values I changed it to.  ¯\_(ツ)_/¯

The only difference I see between the slightly higher and lower values is how much grinding the player will have to do to be able to afford a ship. Or how long they'll have to spend making money elsewhere to do so. Which, by the way, seems weird. Why should how much cargo one has moved in some far off corner of space have any relevance whatsoever to when the Remnant let you take command of one of their ships? I know, it is the conceit of the game and not worth debating, but I'd still rather minimize how much it gets thrown in the player's face.

Debating +/- 100k on a mission seems kind of inconsequential in an area of space where there is currently *only* these three repeating missions, and the cheapest non-specialty ship available is 14.7M credits. Even with the higher values, the player won't earn enough credits through missions alone to buy so much as a Starling by the end of all the missions in Remnant space (repeating jobs not included)

Reply:
>You suggested the values I changed it to. `¯\_(ツ)_/¯`

What I said is that I don't think that other missions need changed because of the addition of these broken JD missions/jobs, and that all you could do is to reduce the payment of broken JD missions to 250k. But in so far as other missions are changed, and in so far as I have been using the difference between the payment for failing and the payment for succeeding Salvage 4 to scale the payment of the broken JD missions, it seems a discrepancy to me to have two missions where you do the exact same thing give differing payments, the exact same thing in this scenario being destroying a `Korath Raider (Hyperdrive)`. If anything, me pointing this out now is actually catching an error in the payment balancing logic of #3761, as that gave 300k for Bounty 2 and 250k for failing Salvage 4.

>the cheapest non-specialty ship available is 14.7M credits. Even with the higher values, the player won't earn enough credits through missions alone to buy so much as a Starling by the end of all the missions in Remnant space (repeating jobs not included)

Most of the credits the player earns are likely going to be through plundering the Korath Raiders, so I don't find the combined payment of all Remnant missions being an issue, but I do think that the Remnant jobs and missions need to remain consistent in the payment they provide for certain tasks. Same task? Same payment.

Reply:
The main point there is intent. The bounty missions are just "deal with the Raider, we don't really care how" vs Salvage 4 where the mission is specifically to disable the Raider. Which is removing options from the player and (depending on their preferred method of combat) making it harder to deal with, hence the higher  pay. And if they fail, they get paid less. In retrospect, I should drop the payment for the failure down to about a third of the payment for success, maybe even down to a quarter.